docs/src/references/c8y-configuration-management.md

```diff
@@ -89,16 +152,68 @@ files = [
   ]
 ```
 
+Along this `c8y_configuration_plugin` configuration for the main device,
+the configuration plugin expect a configuration file per child device
+that needs to be configured from the cloud.
+* The configuration for a child-device `$CHILD_DEVICE_ID`
+  is stored by default under `/etc/tedge/c8y/$CHILD_DEVICE_ID/c8y-configuration-plugin.toml`
+* These TOML files have the same schema as for the main device,
+  listing the configuration `files` and giving for each a `path` and possibly a `type`.
+* Note that the `path` doesn't need to be a file path.
+  It can be a key path in some registry of the child device or any name that makes sense for the child device.
+* As for the main device, the `type` is used to name the configuration file on the cloud.
+  All the configuration `type`s for a child devices are declared to the cloud on start
+  and on change of the `c8y/$CHILD_DEVICE_ID/c8y-configuration-plugin.toml` file.
+* The `user`, `group` and `mode` can be provided for a child-device configuration file
+  but will not be used by the main device if provided.   
```

Comment:
For what reason should a user provide those? Are they sent/provided to the child-device? A short hint would help here.

Reply:
The motivation is to let the child-device send its configuration file unchanged to the main device. I agree this is not clear.

I slightly change the working.